### ISSUE TITLE
Implement an account mangamement system with GitHub auth

### ISSUE THREAD

**Issue by bcardarella** [ElixirMostWanted/elixirmostwanted.com#2]
Because there will be voting and submissions I'd prefer to limit to some public identities rather than anonymous auth.

I don't think we need any information from the individuals beyond GH auth.

users:
  * what ever columns are necessary for GH auth

Depends on #1

**Comment by iagocavalcante** [comment]
Guess this information is enough. What do you think?

```Elixir
%{
          uid: Kernel.inspect(auth.uid),
          email: email,
          avatar: avatar_from_auth(auth)
   }
  ```

**Comment by bcardarella** [comment]
@iagocavalcante are these from GH auth? If so then we don't need email. We just need their GH username and whatever token their oauth provider gives

**Comment by moomerman** [comment]
Is the oauth token required?   Would only need that to do something on their behalf on GitHub.  Minimal requirements probably the username and the UID (in case they change username).

**Comment by bcardarella** [comment]
Probably not, we just need to persist the auth session.

**Comment by bcardarella** [comment]
Closed by #6
